Sources/protoc-gen-swiftgrpc/Generator.swift

```diff
@@ -73,12 +73,30 @@ class Generator {
       //\n
       """)
     
-    for moduleName in ["Foundation", "Dispatch", "SwiftGRPC", "SwiftProtobuf"] {
+    let moduleNames: [String]
+    if options.generateNIOImplementation {
+      moduleNames = [
+        "Foundation",
+        "Dispatch",
```

Comment:
Do we need to import `Dispatch` since we're importing `Foundation`? If so, alphabetical order? 😄

Reply:
We might need these on Linux for the old implementation, but probably not for the NIO one.